platform/x11/os_x11.cpp

```diff
@@ -1797,10 +1797,21 @@ void OS_X11::set_context(int p_context) {
 			classHint->res_name = (char *)"Godot_Editor";
 		if (p_context == CONTEXT_PROJECTMAN)
 			classHint->res_name = (char *)"Godot_ProjectList";
+		if (p_context == CONTEXT_PLAYER)
+			classHint->res_name = (char *)"Godot_Player";
 		classHint->res_class = (char *)"Godot";
 		XSetClassHint(x11_display, x11_window, classHint);
 		XFree(classHint);
 	}
+  
```

Comment:
Note: this line added trailing whitespace, could you try to rebase it out? We're slowly trying to enforce no trailing whitespaces in the repo, though we still need to find how to do it best ;)